_meta/prerequisites.md

```diff
@@ -0,0 +1 @@
+A clear description of the prerequisites for taking this course
```

Comment:
perhaps we need an example here to clarify the format.

Reply:
I've [just](https://github.com/esciencecenter-digital-skills/next-lesson-template/pull/4/commits/bdbcf3a6f931cbb6818124f6f7b1fa4bec40e95f) added examples to all sections :grinning: